packages/clarity-js/types/layout.d.ts

```diff
@@ -151,8 +168,8 @@ export interface DocumentData {
 
 export interface RegionData {
     id: number;
-    visible: number;
-    region: string;
+    state: Interaction;
```

Comment:
>Interaction [](http://example.com/codeflow?start=11&length=11)

Can we call this InteractionState as it's weird to call a "state" property an Interaction type.

Reply:
Sounds good - done.